BetterReads/BetterReads/View Controllers/BookDetailViewController.swift

```diff
@@ -185,6 +192,8 @@ class BookDetailViewController: UIViewController {
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         navigationController?.navigationBar.barStyle = .default
+        navigationController?.navigationBar.tintColor = .black
```

Comment:
What color is the tint on the nav bar? Is it black or white? 
Either delete line 179 or 195 please.